lib/rpmsg/remote_device.c

```diff
@@ -428,11 +428,17 @@ int rpmsg_rdev_create_virtqueues(struct virtio_device *dev, int flags, int nvqs,
 				vring_table[idx].align));
 		}
 
+		vqs[idx] = virtqueue_allocate(ring_info.num_descs);
+		if (vqs[idx] == NULL) {
+			return (ERROR_NO_MEM);
```

Comment:
I think this should be `return RPMSG_ERR_NO_MEM;` to be consistent with the namespace.

Reply:
Thanks will fix.